build-system/tasks/coverage-map/index.js

```diff
@@ -16,22 +16,26 @@
 const argv = require('minimist')(process.argv.slice(2));
 const fs = require('fs').promises;
 const log = require('fancy-log');
+const {buildNewServer} = require('../../server/typescript-compile');
+const {cyan} = require('ansi-colors');
 const {dist} = require('../dist');
 const {installPackages} = require('../../common/utils');
 const {startServer, stopServer} = require('../serve');
 
 let puppeteer;
 let explore;
+let transform;
```

Comment:
Let's remove all of the module-level globals, since they are only used within the functions that initialize them with requires